website/source/docs/builders/profitbricks.html.md

```diff
@@ -0,0 +1,75 @@
+---
+description: |
+    The ProfitBricks builder is able to create images for ProfitBricks cloud.
+layout: docs
+page_title: ProfitBricks Builder
+...
+
+# ProfitBricks Builder
+
+Type: `profitbricks`
+
+The ProfitBricks Builder is able to create virtual machines for [ProfitBricks](https://www.profitbricks.com).
+
+-> **Note:** This builder is not supported by HashiCorp. Please visit [ProfitBricks DevOps Central](https://devops.profitbricks.com/) for support. You may file issues on [GitHub](https://github.com/profitbricks/docker-machine-driver-profitbricks/issues).
```

Comment:
Is the url intended to be https://github.com/profitbricks/docker-machine-driver-profitbricks/issues ?